integration-tests/pom.xml

```diff
@@ -190,6 +195,7 @@
                                 -Dfile.encoding=UTF-8
                                 -Dtestrunfactory=org.testng.DruidTestRunnerFactory
                                 -Ddruid.test.config.dockerIp=${env.DOCKER_IP}
+                                -Ddruid.test.config.hadoopDir=${env.HADOOP_DIR}
```

Comment:
or is this just evaluated in the host env? I'm a bit flummoxed on when this gets executed.